lib/SmartyPlugins/LMSSmartyPlugins.php

```diff
@@ -1152,6 +1152,41 @@ public static function networkDeviceTypesFunction(array $params, $template)
             . '</select>';
     }
 
+    public static function networkDeviceSelectionFunction(array $params, $template)
+    {
+        $LMS = LMS::getInstance();
+
+        static $netdevicelist = array();
+        if (empty($netdevicelist)) {
+            $netdevicelist = $LMS->GetNetDevList();
+        }
+
+        unset($netdevicelist['total'], $netdevicelist['order'], $netdevicelist['direction']);
+
+        $elemname = empty($params['elemname']) ? null : 'name="' . $params['elemname'] . '"';
+        $onchange = empty($params['onchange']) ? null : 'onchange="' . $params['onchange'] . '"';
+        $id = empty($params['id']) ? null : 'id="' . $params['id'] . '"';
+        $selected = empty(intval($params['selected'])) ? null : intval($params['selected']);
```

Comment:
empty(intval(....)) - po co tak zawile? Przecieżm gdy intval() zwróci != `0` to też jest true.